lib/models/message.ts

```diff
@@ -5,12 +5,21 @@ import * as DocumentDbUtils from "../utils/documentdb";
 import { DocumentDbModel } from "../utils/documentdb_model";
 
 import { Option } from "ts-option";
-import { Either } from "../utils/either";
+import { Either, left, right } from "../utils/either";
 import { isNonEmptyString, NonEmptyString } from "../utils/strings";
 
 import { BodyShort, isBodyShort } from "../api/definitions/BodyShort";
 import { FiscalCode, isFiscalCode } from "../api/definitions/FiscalCode";
 
+import {
+  BlobService,
+  getBlobUrl,
+  upsertBlobFromText
+} from "../utils/azure_storage";
+
+const MESSAGE_BLOB_STORAGE_CONTAINER_NAME = "message-content";
```

Comment:
I would pass the name of the container to the `Message` model constructor as a parameter and in turn read the value from the environment from the code that instantiates the `Message` model.